### DISCUSSION TITLE
make the repos private

### DISCUSSION THREAD

**Discussion by akouete-kpakpo** [ML-for-B-E/.github#11] [Q&A]
@BWhannou haven't we said that the repos should be turned to private?

**Comment by BWhannou** [comment]
@akouete-kpakpo Yes, I already turned most of them to private

**Reply by akouete-kpakpo** [reply]
ok I still had this repo Public so I wanted to be sure

![image](https://user-images.githubusercontent.com/26568112/183768136-8ba9c43e-8486-4825-9e94-dae7fb5731ad.png)
